packages/cashier/src/pages/withdrawal/crypto-withdraw-receipt/__tests__/crypto-withdraw-receipt.spec.tsx

```diff
@@ -1,6 +1,7 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
@george-usynin-binary This will be fixed [here](https://github.com/binary-com/deriv-app/pull/6306/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R64-R67), No need to ignore it.